commands/service_execute.go

```diff
@@ -50,49 +50,65 @@ func (c *serviceExecuteCmd) preRunE(cmd *cobra.Command, args []string) error {
 }
 
 func (c *serviceExecuteCmd) runE(cmd *cobra.Command, args []string) error {
-	s, err := c.e.ServiceByID(args[0])
-	if err != nil {
-		return err
-	}
+	var (
+		err            error
+		listenResultsC chan *coreapi.ResultData
+		resultsErrC    chan error
+	)
+	pretty.Progress(fmt.Sprintf("Executing task %q...", c.taskKey), func() {
```

Comment:
Do we really need all those calls inside progress? let's put just ExecuteTask call.

Reply:
Yes because actually, the CLI call Core for: `ServiceByID`, `ServiceListenResults` and `ServiceExecuteTask`. So I group everything into only one loader

Reply:
ok